py/selenium/webdriver/remote/webdriver.py

```diff
@@ -157,9 +157,7 @@ def create_matches(options: List[BaseOptions]) -> Dict:
                     if opts[min_index][kys] == opts[i + 1][kys]:
                         samesies.update({kys: opts[min_index][kys]})
 
-    always = {}
-    for k, v in samesies.items():
-        always[k] = v
+    always = samesies.copy()
```

Comment:
copy is likely better performance than the comprehension counter part; however is a _shallow_ copy sufficient here? (I'd need to look at the code, no time right now)

Reply:
Sure, letme know if it can be improved. I also saw that this new local variable isnt much used so maybe it can be completely removed while using samesies variables but i need to look at code if it messes up something

Reply:
Since always is a local variable of the function used in 3 places maybe this unnecessary dict copy can be replaced by samesies. 
```
always = {}
    for k, v in samesies.items():
        always[k] = v

    for i in opts:
        for k in always.keys():
            del i[k]

    capabilities["capabilities"]["alwaysMatch"] = always
```
To
```
#always = {}
#    for k, v in samesies.items():
#       always[k] = v

    for i in opts:
        for k in samesies.keys():
            del i[k]

    capabilities["capabilities"]["alwaysMatch"] = samesies
```

Reply:
@symonk

Reply:
I think the later is probably what we want